modules/apps/object/object-test/src/testFunctional/paths/Picklist.path

```diff
@@ -155,6 +155,16 @@
 		<td>//div[contains(@class, 'dropdown')]//button//*[name()='svg'][contains(@class,'download')]</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>SELECT_PICKLIST_AUTO_COMPLETE</td>
+		<td>//button[contains(@class, 'form-base-picklist-select-field')]</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>SELECT_PICKLIST</td>
+		<td>//div[contains(@class, 'dropdown-menu-select') and contains(., 'Picklist')]</td>
+		<td></td>
+	</tr>
```

Comment:
The way to select a picklist has changed and we don't have any generic path that we can use.